app/models.py

```diff
@@ -1495,15 +1488,15 @@ def unsubscribe_link(self, contact: Optional["Contact"] = None) -> (str, bool):
         The mailto: method is preferred
         """
         if contact:
-            if UNSUBSCRIBER:
-                return f"mailto:{UNSUBSCRIBER}?subject={contact.id}_", True
+            if config.UNSUBSCRIBER:
+                return f"mailto:{config.UNSUBSCRIBER}?subject={contact.id}_", True
```

Comment:
Should we use the UnsubscribeEncoder.encode here?

Reply:
I was leaving it for the next but this can simplified for this one. I've removed this function and moved the logic to the encoder.